rules/filename-case.js

```diff
@@ -3,6 +3,7 @@ var path = require('path');
 var camelCase = require('lodash.camelcase');
 var kebabCase = require('lodash.kebabcase');
 var snakeCase = require('lodash.snakecase');
+var pascalCase = require('pascal-case');
```

Comment:
This brings in a whole bunch of duplicate dependencies. Instead let's stick with lodash:

```js
var capitalize = require('lodash.capitalize');

var pascalCase = function(string) {
  return capitalize(camelCase(string));
};